packages/jest-editor-support/src/__tests__/parsers/parserTests.js

```diff
@@ -19,6 +19,10 @@ function parserTests(parse: (file: string) => ParserReturn) {
     it('For the simplest it cases', () => {
       const data = parse(`${fixtures}/global_its.example`);
 
+      if (data.skip) {
```

Comment:
revert this, the tests should be run because typescript is a devDependency of the root package.json.